src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -1119,3 +1121,129 @@ def convert_api_span_for_insertion(api_span: Span) -> SpanForInsertion:
+        If the deleted span has a parent, negative cumulative values will be propagated
+        up the ancestor chain.
+        Note: This operation is irreversible.
+        """
+    ),
+    responses=add_errors_to_responses([HTTP_404_NOT_FOUND]),
+    status_code=204,  # No Content for successful deletion
+)
+async def delete_span(
+    request: Request,
+    project_identifier: str = Path(
+        description=(
+            "The project identifier: either project ID or project name. If using a project name, "
+            "it cannot contain slash (/), question mark (?), or pound sign (#) characters."
+        )
+    ),
+    span_identifier: str = Path(
+        description="The span identifier: either a relay GlobalID or OpenTelemetry span_id"
+    ),
+) -> None:
+    """
+    Delete a span by span_identifier within a project, leaving all descendants alone.
+
+    1. Find the project by identifier
+    2. Find the target span to delete
+    3. Delete only the target span (leave descendants alone)
+    4. Check if trace is empty - if so, delete the trace record
+    5. If span had a parent, propagate negative cumulative values up ancestor chain
+    6. Return 204 No Content on success
+    """
+    async with request.app.state.db() as session:
+        project = await _get_project_by_identifier(session, project_identifier)
+        project_id: int = project.id
+
+        # Try to parse as GlobalID first, then fall back to span_id
+        try:
+            span_rowid = from_global_id_with_expected_type(
+                GlobalID.from_id(span_identifier),
+                "Span",
+            )
+            # Find by database rowid and verify it belongs to the project
+            target_span = await session.scalar(
+                select(models.Span)
+                .join(models.Trace, models.Trace.id == models.Span.trace_rowid)
+                .where(models.Trace.project_rowid == project_id)
+                .where(models.Span.id == span_rowid)
+            )
+            error_detail = (
+                f"Span with relay ID '{span_identifier}' not found in "
+                f"project '{project_identifier}'"
+            )
+        except Exception:
+            # Find by OpenTelemetry span_id
+            target_span = await session.scalar(
+                select(models.Span)
+                .join(models.Trace, models.Trace.id == models.Span.trace_rowid)
+                .where(models.Trace.project_rowid == project_id)
+                .where(models.Span.span_id == span_identifier)
+            )
+            error_detail = (
+                f"Span with span_id '{span_identifier}' not found in project '{project_identifier}'"
+            )
+        if target_span is None:
+            raise HTTPException(
+                status_code=HTTP_404_NOT_FOUND,
+                detail=error_detail,
+            )
+
+        # Store values needed for later operations
+        trace_id = target_span.trace_rowid
```

Comment:
the convention is to use rowid
```suggestion
        trace_rowid = target_span.trace_rowid
```